spec/koans.js

```diff
@@ -8,31 +8,32 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+        varX= true;
       }
-      // expect(varX).toBe(true);
+      expect(varX).toBe(true);
+    
     });
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
+      var letX = false
       if (true) {
-        /*var or const? letX = true*/
+        const letX = true
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
     });
 
     it('`var` does not restricts scope to inside the block in `for` loops', () => {
-      /*var or let? counter = 100*/
-      /*for (var or let? counter = 1; counter < 50; counter++){}*/
+      counter = 100;
```

Comment:
You can use var here, even if you re-declare this variable inside the loop, the same variable will just have its value reassigned.